test/testdata/infer/overloads_test.rb

```diff
@@ -16,25 +16,25 @@ def make_untyped
   end
   sig do
     params(
-      _: String,
+      _0: String,
     )
     .returns(String)
   end
   sig do
     params(
-        _: Exception,
+        _0: Exception,
```

Comment:
Now a syntax error

Reply:
to check my understanding, it would have been sufficient to fix this syntax error by  changing

```ruby
  def overloaded(_=_, _1=_, _2=_);	
```

to

```ruby
  def overloaded(_=_0, _1=_, _2=_);	
```

right?

(I think the current change is fine, just wanted to make sure I understand it)

Reply:
Yes, exactly.

I updated the pull-request to use you example so it's explicit now.